examples/ethtool_coalesce.yml

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ network_interface_name1 }}"
+            state: up
+            type: ethernet
+            ip:
+              dhcp4: "no"
+              auto6: "no"
```

Comment:
Why are those values quoted? IIUC, those are meant to be boolean values, not literal "no" strings.

Reply:
I think the yamllint truthy check wanted this in the past.

Reply:
Because it was a copy-paste from examples/ethtool_features.yml :) I removed them.

Reply:
> I think the yamllint truthy check wanted this in the past.

I see. Yes, this check used to be a problem, but it has improved recently and you can tell yamllint what values for booleans do you want to accept. So you can configure it to accept the usual Ansible convention of `yes`/`no`/`true`/`false` (convention decided here: https://meetbot.fedoraproject.org/ansible-docs/2019-01-08/docs_working_group.2019-01-08-15.31.html ). I submitted a PR for this: https://github.com/linux-system-roles/template/pull/34. That said, `on`/`off` do not conform to the convention either :-) but let's leave it for a separate PR.